playground/mocks/data/idp/idx/identify-with-user-verification-loopback.json

```diff
@@ -0,0 +1,177 @@
+{
+  "stateHandle": "02im-3M2f6UXHgNfS7Ns7C85EKHzGaKw0u1CC4p9_r",
+  "version": "1.0.0",
+  "expiresAt": "2020-01-13T21:14:37.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "relatesTo": [
+          "$.currentAuthenticatorEnrollment"
+        ],
+        "name": "user-verification-poll", 
```

Comment:
@santoshmale-okta is the name okay with you?

Reply:
we can probably just name it as `challenge-authenticator` to be consistent with all other factor challenges on the IDX pipeline.